sdk/metrics-testing/src/main/java/io/opentelemetry/sdk/testing/assertj/metrics/ExponentialHistogramAssert.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.testing.assertj.metrics;
+
+import io.opentelemetry.sdk.metrics.data.AggregationTemporality;
+import io.opentelemetry.sdk.metrics.data.ExponentialHistogramData;
+import io.opentelemetry.sdk.metrics.data.ExponentialHistogramPointData;
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.AbstractIterableAssert;
+import org.assertj.core.api.Assertions;
+
+/** Test assertions for {@link ExponentialHistogramData}. */
+public class ExponentialHistogramAssert
+    extends AbstractAssert<ExponentialHistogramAssert, ExponentialHistogramData> {
+
+  protected ExponentialHistogramAssert(ExponentialHistogramData actual) {
+    super(actual, ExponentialHistogramAssert.class);
+  }
+
+  /** Ensures that {@code aggregation_temporality} field is {@code CUMULATIVE}. */
+  public ExponentialHistogramAssert isCumulative() {
+    isNotNull();
+    if (actual.getAggregationTemporality() != AggregationTemporality.CUMULATIVE) {
+      failWithActualExpectedAndMessage(
+          actual,
```

Comment:
We'll need to add test cases for this class - should we file an issue for it or is it simple enough now? Or can we delete the class for now if it's not useful?

Reply:
I've gone ahead and added tests for it. Codecov seems to be green now.